lambdas/ci-mapping/src/ci-mapping-event-validator.ts

```diff
@@ -16,6 +16,7 @@ export interface CiMappingEvent {
   contraIndicationMapping: string[];
   hmrcErrors: string[];
   contraIndicatorReasonsMapping: Array<CiReasonsMapping>;
+  govJourneyId: string;
```

Comment:
I can't see this used any where in this code?

Reply:
CiMappingHandler: 
```
event.govJourneyId,
```